src/actions/asyncActions.ts

```diff
@@ -119,6 +126,58 @@ export const AsyncActions = {
             dispatch(Actions.updateRssPosts(posts));
         };
     },
+    downloadPrivatePostsFromContacts: (contactFeeds: ContactFeed[]): Thunk => {
```

Comment:
Is there a way to refresh when you are on a view with a specific 1to1 feed?

Reply:
You can pass in an array with one element.